tests/test_quota.py

```diff
@@ -90,8 +90,7 @@ def test_marathon_group_enforced(create_group, setup_spark, group, enforce_group
 
 @pytest.mark.sanity
 @pytest.mark.parametrize('role,enforce_role', [
-    ("spark_role", False),
-    ("custom_role", False)
+    ("dev", False)
```

Comment:
these use cases are still valid for DCOS versions prior to 2.0. Can we somehow refactor the test and mark it to be run with all roles only on DCOS < 2.0?

It looks like we can separate two use-cases:
- _any_ role propagation (DCOS pre-2.0)
- group role propagation (DCOS 2.0 and higher)